custom_components/hacs/helpers/classes/repository.py

```diff
@@ -402,8 +402,10 @@ async def remove_local_directory(self):
 
             if os.path.exists(local_path):
                 if not is_safe_to_remove(local_path):
+                    self.logger.error(f"Path {local_path} is not save to remove")
                     return False
-                self.logger.debug(f"Removing {local_path}")
+                else:
```

Comment:
This change makes absolutely no sense, revert it.
It does the exact same thing without the else here.